src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -160,6 +161,16 @@ public static Configuration mapJdbcCatalogToConfig(Configuration ingestConfig, S
         return newConfig;
     }
 
+  private static Predicate<Schema> matchDatabaseOrSchema(Jdbc jdbc) {
```

Comment:
In mysql,  database and schema both are synonym.  And in oracle or snowflake both database and schema exists. Hence this method is used just to filter schema based on jdbc connection type.